lib/autokey/gtkui/data/phrasepage.xml

```diff
@@ -12,15 +12,43 @@
         <property name="can_focus">False</property>
         <property name="spacing">5</property>
         <child>
-          <object class="GtkLinkButton" id="linkButton">
-            <property name="label" translatable="yes">(Unsaved)</property>
-            <property name="use_action_appearance">False</property>
+          <object class="GtkHBox" id="linkBox">
             <property name="visible">True</property>
-            <property name="can_focus">True</property>
-            <property name="receives_default">True</property>
-            <property name="has_tooltip">True</property>
-            <property name="use_action_appearance">False</property>
-            <property name="relief">none</property>
+            <property name="can_focus">False</property>
+            <child>
+              <object class="GtkLinkButton" id="linkButton">
+                <property name="label" translatable="yes">(Unsaved)</property>
+                <property name="use_action_appearance">False</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">True</property>
+                <property name="has_tooltip">True</property>
+                <property name="use_action_appearance">False</property>
+                <property name="relief">none</property>
+              </object>
+              <packing>
+                <property name="expand">True</property>
+                <property name="fill">True</property>
+                <property name="position">0</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLinkButton" id="linkButton1">
```

Comment:
Can we please use a name other than `linkButton1` for the id? Way too hard to know what it refers to elsewhere